specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json

```diff
@@ -0,0 +1,3470 @@
+            "name": "skuId",
+            "in": "query"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/SkuAvailabilityListResponse"
+            }
+          },
+          "default": {
+            "description": "BadRequest",
+            "schema": {
+              "$ref": "#/definitions/CSRPError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ListAvailabilities": {
+            "$ref": "./examples/ListAvailabilities.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.VMwareCloudSimple/locations/{regionId}/operationResults/{operationId}": {
+      "get": {
+        "description": "Return an async operation",
+        "tags": [
+          "operation"
+        ],
+        "summary": "Implements get of async operation",
+        "operationId": "getOperationResultByRegion",
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/regionId"
+          },
+          {
+            "$ref": "#/parameters/referer"
+          },
+          {
+            "type": "string",
+            "description": "operation id",
+            "name": "operationId",
+            "in": "path",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation",
+            "schema": {
+              "$ref": "#/definitions/OperationResource"
+            }
+          },
+          "202": {
+            "description": "Operation is still running",
+            "headers": {
+              "Location": {
+                "type": "string"
+              },
+              "Retry-After": {
+                "type": "integer"
+              }
+            }
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "BadRequest",
+            "schema": {
+              "$ref": "#/definitions/CSRPError"
+            },
+            "headers": {
+              "Content-Type": {
+                "type": "string",
+                "default": "application/json"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetFailedOperationResult": {
+            "$ref": "./examples/GetFailedOperationResult.json"
+          },
+          "GetOperationResult": {
+            "$ref": "./examples/GetOperationResult.json"
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
This is a GET operation to retrieve operation status of LRO, ```x-ms-long-running-operation``` annotation is not valid for this operation, please remove this annotation.

Reply:
java codegen has a check for incorrect use of annotations, so after removing this annotation, please merge my PR that adds java tag https://github.com/tenmozes/azure-rest-api-specs/pull/1

Reply:
@anuchandy thank you for a review, fixed